### PR TITLE
[Svace] fix svace issues

### PR DESCRIPTION
**Pull request by jaeyun-jung**
update files to fix svace issues

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/capi/src/nnstreamer-capi-util.c [hunk 1]
```diff
@@ -79,7 +79,7 @@ ml_util_validate_tensor_info (const ml_tensor_info_s * info)
   if (!info)
     return ML_ERROR_INVALID_PARAMETER;
 
-  if (info->type < 0 || info->type >= ML_TENSOR_TYPE_UNKNOWN)
+  if (info->type == ML_TENSOR_TYPE_UNKNOWN)
```
Comment (myungjoo) [inline api/capi/src/nnstreamer-capi-util.c]: Is it safe to assume that ```type > ML_TENSOR_TYPE_UNKNOWN``` won't happen?

Comment (jaeyun-jung) [reply api/capi/src/nnstreamer-capi-util.c]: I will change to switch-case to avoid this.

File: api/capi/src/nnstreamer-capi-util.c [hunk 2]
```diff
@@ -79,8 +79,22 @@ ml_util_validate_tensor_info (const ml_tensor_info_s * info)
...
+    case ML_TENSOR_TYPE_INT32:
+    case ML_TENSOR_TYPE_UINT32:
+    case ML_TENSOR_TYPE_INT16:
+    case ML_TENSOR_TYPE_UINT16:
+    case ML_TENSOR_TYPE_INT8:
+    case ML_TENSOR_TYPE_UINT8:
+    case ML_TENSOR_TYPE_FLOAT64:
+    case ML_TENSOR_TYPE_FLOAT32:
+    case ML_TENSOR_TYPE_INT64:
+    case ML_TENSOR_TYPE_UINT64:
```
Comment (myungjoo) [inline api/capi/src/nnstreamer-capi-util.c]: SVACE will complain a lot for this.. :)

**Comment by again4you** [comment]
### Analysis Report
* Result: None (Ready to merge!)
* Link: http://10.113.170.22:8810/download/11543/artifacts/analysis_report_I443d4d_svace-2.6.1-gbs.html

**Pull request merged**